sdk/translation/ai-translation-text-rest/test/public/utils/recordedClient.ts

```diff
@@ -2,123 +2,28 @@
 // Licensed under the MIT license.
```

Comment:
Content of this file, seems to be custom. We might need to manually merge changes - take all from original file and maybe add some funtionality from the new generated file.
We might also need to disable tests creation during the script execution if possible.

Reply:
Reverted.  Now the tests are running.

Reply:
Resolve this comment after you will make tests creation disabled for code gen.